dev/collect_upstream_conda_tests/collect_upstream_conda_tests.py

```diff
@@ -25,6 +25,10 @@
         # Features / nomkl involved
         "test_features_solve_1",
         "test_prune_1",
+        "test_update_prune_2",
+        "test_update_prune_3",
+        # Message expected, but libmamba does not report constraints
+        "test_update_prune_5",
```

Comment:
Is this something we can tweak in the upstream test to work better here?

Reply:
Not really. libmamba's implementation doesn't really report that kind of thing so if anything we have to add more messaging to conda-libmamba-solver wrapping logic.